sympy/core/symbol.py

```diff
@@ -205,10 +205,22 @@ def numbered_string_incr(s, start=0):
 
 class Symbol(AtomicExpr, Boolean):
     """
+    A Symbol is a representation of a mathematical variable using a
+    name. It supports subscripts and Greek and Latin characters.
+    Symbols can be combined to form SymPy expressions. A single
+    Symbol can be created using the Symbol() constructor but to create
+    multiple symbols in one statement use the symbols() function.
+
     Assumptions:
        commutative = True
 
+    There are more assumptions for a Symbol whose default value is set to
+    ``None``. To read more about them see
+    [https://docs.sympy.org/latest/guides/assumptions.html#predicates](url)
```

Comment:
It would be better to use a sphinx reference here rather than an absolute URL.

Reply:
I have changed it. Could you confirm whether it has been done correctly?

Reply:
Why would this still need URL?
If the target is inside the page, it could be possible to reference without URL.

https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html

Reply:
Is the target in the same page because I cannot seem to find it?

Reply:
You'd need to check out `` `predicates`_ `` target work

Reply:
Hi I don't quite think this target exists as I did try replacing the url with ``:ref: `predicates`_``. However, an error was raised saying that such a target does not exist. Should I revert back to the original syntax or have I done something wrong?